src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -365,7 +365,7 @@ export default {
   },
 
   computed: {
-    ...mapGetters(['isBComp', 'isRoleStaff', 'nrNumber']),
+    ...mapGetters(['getIncorporationNumber', 'isBComp', 'isRoleStaff', 'nrNumber']),
```

Comment:
Incorporation Number is the same as Entity Inc No... let's stick with a single property :)

Maybe add a comment where this is used if it helps code clarity.

Reply:
Hmm, ok good point. This is probably a duplicate then, i will stick with the original Entity Inc no. Will fix!

Reply:
So this was just poor naming convention. I created a getter to fetch entityIncNo.. and called it something different :P 
Fixed!

Reply:
Would you mind replacing "entityIncNo" with your new getter in this file?

Reply:
Absolutely!